### DISCUSSION TITLE
revit to imodel

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#159] [Q&A] [answered]
now i have a main rvt file ,the main file have some refance rvt file, i use  revit bentley plugin menu to export one  i.bim file
in imodeljs ,can  not find elements in the refance rvt file ( can display,i find the refance file show but is indivisible)

**Comment by Rickzzzzzhai** [comment]
Can you try to bind your refence rvt file to the main rvt file (sorry I forgot the function name in Revit)? Then export a i.bim file and I think you can find all elements. 
Hope this helps.

**Comment by abeesh** [comment, marked as answer]
If you have two Revit files, you can use iTwin synchronizer to publish both of them into the same iModel. If the coordiantes in both these models are correct, then they will be presented correctly inside the iModel.
